app/views/events/index.html.erb

```diff
@@ -25,6 +25,11 @@
         <td><%= link_to 'Show', event %></td>
         <td><%= link_to 'Edit', edit_event_path(event) %></td>
         <td><%= link_to 'Attend', attend_path(event) %></td>
+        <% if event.creator_id == @current_user.id %>
+          <td><%= button_to 'Destroy', event, method: :delete,data: { confirm: 'Are you sure?' }, class: 'btn btn-danger'%></td>
```

Comment:
- Add a space after `:delete`, i.e.  
```ruby
<%= button_to 'Destroy', event, method: :delete, data: { 
```

- Maybe "Are you sure to delete this event?" would be a better confirm message.

Reply:
- I made the modification on the spacing
- for the front part, here is my suggestion for the display of an event on the page where all the events will be displayed
![event](https://user-images.githubusercontent.com/52690513/114407962-a08c5480-9bb1-11eb-85dd-837ebea8652c.png)

Reply:
the word "avis" will be changed to "review", and the follow icon will be modified.

Reply:
OK, lets roll with that frontend

Reply:
ok,